packages/user-ui/src/pages/projects/_id/discussion/_id/page.rs

```diff
@@ -47,19 +49,20 @@ pub fn DiscussionVideoPage(
                     }
                 }
 
-                //FIXME: fix to real email
                 ParticipantSidebar {
                     show_member: show_side_member(),
                     hide_member: move |_| {
                         show_side_member.set(false);
                     },
-                    emails: vec![
-                        "aaa@bbb.ccc".to_string(),
-                        "aaa1@bbb.ccc".to_string(),
-                        "aaa2@bbb.ccc".to_string(),
-                        "aaa3@bbb.ccc".to_string(),
-                        "aaa4@bbb.ccc".to_string(),
-                    ],
+                    refresh: move |_| {
+                        ctrl.refresh();
+                    },
+
+                    clicked_attendee: move |attendee_id: String| {
+                        ctrl.clicked_attendee(attendee_id);
+                    },
```

Comment:
```suggestion
                    onselect: move |attendee_id: String| {
                        ctrl.handle_selecting_attendee(attendee_id);
                    },
```
이런식으로 이름이 구성되는게 조금더 맞습니다.

그런데 왜 여기에서는 attendee_id가 string인가요? chime에서 제공해주는 id라서 그런가요?

Reply:
수정했고, attendee_id는 chime에서 제공해주는 id 라 string입니다.